packages/scripts/src/commands/app-data/translate.ts

```diff
@@ -0,0 +1,64 @@
+import path from "path";
+import fs from "fs-extra";
+import chalk from "chalk";
+import { spawnSync } from "child_process";
+import { logError, recursiveFindByExtension } from "../../utils";
+import { getActiveDeployment } from "../deployment/get";
+import { Command } from "commander";
+
+/***************************************************************************************
+ * CLI
+ * @example yarn
+ *************************************************************************************/
+const program = new Command("translate");
```

Comment:
code related to applying translations has mostly been separated out into its own scripts